src/service/poc_mobile.proto

```diff
@@ -616,3 +621,46 @@ message oracle_boosting_hex_assignment {
   // Landtype assignment
   assignment landtype = 5;
 }
+
+message service_provider_boosted_rewards_banned_radio_req_v1 {
+  enum sp_boosted_rewards_banned_radio_reason {
+    unbanned = 0;
+    no_network_correlation = 1;
+  }
+  // pubkey of signer
+  bytes pub_key = 1;
```

Comment:
i prefer pubkey but i see both are in use throughout the protos

Reply:
Fixed [8f9a428](https://github.com/helium/proto/pull/405/commits/8f9a42803fef0efd0e2fe5d14687d88c48cbbea3)